src/interfaces/cb-types.ts

```diff
@@ -62,3 +63,8 @@ export type Addon = {
   id?: string;
   quantity?: number;
 };
+
+export type Item = {
+  planPricePointId?: string;
```

Comment:
should this be `itemPricePointId` instead of `planPricePointId` ? And also make it mandatory?

Reply:
Made it mandatory. Name is kept as planPricePointId as the docs suggest the name and in UI item are referred as plans